### PR TITLE
Remove MetaStation.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
## About The Pull Request

This pull request removes metastation from the game. 

## Why It's Good For The Game
People compare every map to metastation, Complain when ever a map does things different to meta station, call other maps not having something metastation has a bug, and basically every map is judged by how (not) different it is to meta station.

As somebody who made the map rotation system purely so meta station could succeed and get more play time, even i can see how cancerous this is.

## Changelog
:cl:
del: Removes metastation.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
good god

**Comment by ShibaInuLord** [comment]
MSO going ham

File: config/maps.txt [hunk 1]
```diff
@@ -13,13 +13,6 @@ Format:
 	votable (is this map votable)
 endmap
 
-map metastation
-	#default
```
Comment (Jared-Fogle) [inline config/maps.txt:17]: No more default map :tada:

**Comment by BigGrease** [comment]
no thanks

**Comment by justinbooj9** [comment]
ah yes remove metastation but not pubby. 10/10 best pr

**Comment by MMMiracles** [comment]
i could of used this about a year ago

**Review by wesoda25 — APPROVED** [review]

**Review by san7890 — APPROVED** [review]

**Comment by notfrying1pans** [comment]
shut the fuck up remove pubby instead

**Comment by msgerbs** [comment]
I just spent 8 hours remodeling the brig on Meta...

**Comment by ArcaneMusic** [comment]
...Okay, alright, even if I DID AGREE with this, at least attach the issues that this would close alongside the PR so we can see the impact of removing this.

**Comment by Mickyan** [comment]
Gonna take the bait and point out that removing something for the sole reason that people (perhaps incorrectly) use it as an example of something done right is incredibly obtuse

The map rotation was already changed to give other maps a chance to be played and did so with great success, take it a step further and force more variety if that's the issue

**Comment by Cyberboss** [comment]
This was a good PR tbh. Can't believe box died for this mess

**Comment by msgerbs** [comment]
Box didn't die, it just isn't in space anymore.

**Comment by BigGrease** [comment]
how did it get onto hoth

**Comment by YakiAttaki** [comment]
was this a joke or

**Pull request closed without merging**